{"uuid":"7556a5bb-ebf4-41b5-a1f8-540f5e8e3d3e","historyId":"d6cf88bc1e6a5b7a299f750253f31de2","fullName":"src/test/resources/features/05_Array.feature:300","labels":[{"name":"tag","value":"Array"},{"name":"tag","value":"TS_array_35"},{"name":"host","value":"Anjus-Air"},{"name":"thread","value":"64609@Anjus-Air.main(1)"},{"name":"feature","value":"Array feature"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Squares of a Sorted Array"},{"name":"suite","value":"Array feature"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Squares of a Sorted Array"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///Users/anjujohn/eclipse-workspace5/DS_Algo_Project/src/test/resources/features/05_Array.feature"},{"name":"package","value":"src.test.resources.features.05_Array_feature.Array feature"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Squares of a Sorted Array","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on Question page of \"Squares of a Sorted Array\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704234091086,"stop":1704234091197},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704234091198,"stop":1704234091477},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704234091478,"stop":1704234091593},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704234091595,"stop":1704234091613}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1704234091084,"stop":1704234091614}